src/gui/widgets/Knob.cpp

```diff
@@ -689,10 +689,48 @@ void Knob::paintEvent( QPaintEvent * _me )
 void Knob::wheelEvent(QWheelEvent * we)
 {
 	we->accept();
-	const float stepMult = model()->range() / 2000 / model()->step<float>();
-	const int inc = ((we->angleDelta().y() > 0 ) ? 1 : -1) * ((stepMult < 1 ) ? 1 : stepMult);
-	model()->incValue( inc );
+	const int deltaY = we->angleDelta().y();
+	float direction = deltaY > 0 ? 1 : -1;
+
+	auto * m = model();
+	float const step = m->step<float>();
+	float const range = m->range();
+
+	float numberOfStepsForFullSweep = 100.;
+
+	auto const modKeys = we->modifiers();
+	if (modKeys == Qt::ShiftModifier)
+	{
+		// With shift we want to be able to go through the values in 10 steps
+		numberOfStepsForFullSweep = 10;
+	}
+	else if (modKeys == Qt::ControlModifier)
+	{
+		// With control we want to be able to go through the values in 100 steps
```

Comment:
100 steps? This and the next comment does not look up to date.

Reply:
I have adjusted the comments by better describing the ideas instead of referencing wrong values. 😄